project/src/const.js

```diff
@@ -0,0 +1,9 @@
+export const AppRoute = {
+  SING_IN: '/login',
```

Comment:
Здесь и в других местах

```suggestion
  SIGN_IN: '/login',
```